tools/topology/topology2/doc/build/sof.doxygen

```diff
@@ -0,0 +1,39 @@
+PROJECT_NAME     = "Sound Open Firmware Topology2"
+OUTPUT_DIRECTORY = doxygen
+GENERATE_LATEX   = NO
+GENERATE_RTF	 = NO
+GENERATE_MAN     = NO
+GENERATE_XML	 = YES
+
+CASE_SENSE_NAMES = NO
+INPUT            = /home/oku/work/sof-zephyr/sof/doc/../tools/topology/topology2
```

Comment:
this will need to be relative.

Reply:
Oh. That should not be there at all. Its produced by cmake.